docker/Dockerfile

```diff
@@ -47,12 +49,23 @@ RUN wget -q -P /tmp \
     && bash /tmp/Miniconda3-latest-Linux-x86_64.sh -b -p /opt/conda \
     && rm /tmp/Miniconda3-latest-Linux-x86_64.sh
 
+ARG OS=ubuntu2004
```

Comment:
Are the devel libraries necessary?
To avoid too many docker file layers, how about changing this to:

```
ARG OS=ubuntu2004
RUN wget -q https://developer.download.nvidia.com/compute/cuda/repos/${OS}/x86_64/cuda-${OS}.pin \
  -O /etc/apt/preferences.d/cuda-repository-pin-600 \
  && apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/${OS}/x86_64/7fa2af80.pub \
  && add-apt-repository "deb https://developer.download.nvidia.com/compute/cuda/repos/${OS}/x86_64/ /" \
  && apt-get update \
  && apt-get install libcudnn8 libcusolver-${CUDA/./-} 
```

Reply:
OK, I will remove the dev libraries.